client/src/Pages/Home/index.js

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from 'react';
+import Typography from '@mui/material/Typography';
+import { Link } from 'react-router-dom';
+import { experimentalStyled as styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Grid from '@mui/material/Grid';
+import Footer from '../../Components/Common/Footer';
+import CustomersOpinion from '../../Components/Common/CustomersOpinion';
+import ImageCard from '../../Components/Common/ImageCard';
+import Header from '../../Components/Common/Header';
+import SearchBox from '../../Components/Common/SearchBox';
+import NavBar from '../../Components/Common/Navbar';
+import './style.css';
+
+const axios = require('axios');
+
+function Home() {
+  const [brands, setBrands] = useState([]);
+  const [types, setTypes] = useState([]);
+  const [review, setReview] = useState([]);
+  const getCarsBrands = async () => {
+    try {
+      const data = await axios.get('/api/v1/brands');
+      setBrands(data.data.data);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  const getTCarsType = async () => {
+    try {
+      const data = await axios.get('/api/v1/types');
+      setTypes(data.data);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  useEffect(() => {
+    getTCarsType();
+  }, []);
```

Comment:
all your effects need a cleanup, try to find a way to clean up Axios REST call.

Reply:
done